www/src/components/rotater.js

```diff
@@ -0,0 +1,112 @@
+import React, { Component } from "react";
+import Slider from "./Slider";
+import { rhythm, options } from "../utils/typography"
+import presets, { colors, space } from "../utils/presets"
+import Link from "gatsby-link"
+
+class Search extends Component {
+  constructor(props, context) {
+    super(props, context)
+    this.state = { item: 0 }
+  }
+
+  decrementItem() {
+    clearInterval(this.state.intervalId);
+    this.setState({
+      intervalId: -1, 
+      item: ((this.state.item + this.props.items.length - 1) % this.props.items.length)
+    })
+  }
+  incrementItemAndClearInterval() {
+    clearInterval(this.state.intervalId);
+    this.incrementItem()
+    this.setState({
+      intervalId: -1,
+    })
+  }
+  
+  incrementItem() {
+    this.setState({ 
+      item: ((this.state.item + 1) % this.props.items.length)
+    })
+  }
+  componentDidMount() {
+    const intervalId = setInterval(() => this.incrementItem(), 5000);
```

Comment:
I'm wondering if we should work with onAnimationEnd event instead of an interval? If we keep an interval I would suggest to use setTimeout instead as setInterval might not be 5s over time and setTimeout always will be.